bug: on 32GB CPU and 1GB GPU, Jan fails to load 4B model

**Version:** 0.7.9

## Describe the Bug

Jan thinks that GPU/CUDA is the best backend when the GPU has only 1GB VRAM, downloads 4B model, which fails to run, because it doesn't fit.

It should use 32GB CPU RAM instead in this case.

## Steps to Reproduce
1. Download Jan
2. Wait for it to download model
3. Try to chat

## Screenshots / Logs
```
[2026-04-01][06:05:30][tauri_plugin_llamacpp::backend][INFO] Determined best available backend: b8149/win-avx-cuda-cu11.7-x64 (Category: "cuda-cu11.7")
...
[2026-04-01][06:18:58][tauri_plugin_llamacpp::gguf::commands][INFO] modelSize: 2999182176
[2026-04-01][06:18:59][tauri_plugin_llamacpp::gguf::commands][INFO] Using ctx_size: 8192
[2026-04-01][06:18:59][tauri_plugin_llamacpp::gguf::utils][INFO] Received ctx_size parameter: Some(8192)
[2026-04-01][06:18:59][tauri_plugin_llamacpp::gguf::utils][INFO] KV estimate (no SWA detected) -> full: 1207959552 bytes (~1152.00 MB)
[2026-04-01][06:18:59][tauri_plugin_llamacpp::gguf::commands][INFO] isModelSupported: Total memory requirement: 4207141728 for \\?\C:\Users\Администратор\AppData\Roaming\Jan\data\llamacpp\models\Jan-v3.5-4B-Q4_K_XL\model.gguf; Got kvCacheSize: 1207959552 from BE
[2026-04-01][06:18:59][tauri_plugin_llamacpp::gguf::commands][INFO] Total VRAM reported/calculated (in bytes): 1073741824
[2026-04-01][06:18:59][tauri_plugin_llamacpp::gguf::commands][INFO] System RAM: 34278998016 bytes
[2026-04-01][06:18:59][tauri_plugin_llamacpp::gguf::commands][INFO] Total VRAM: 1073741824 bytes
[2026-04-01][06:18:59][tauri_plugin_llamacpp::gguf::commands][INFO] Usable total memory: 31990507827 bytes
[2026-04-01][06:18:59][tauri_plugin_llamacpp::gguf::commands][INFO] Usable VRAM: 0 bytes
[2026-04-01][06:18:59][tauri_plugin_llamacpp::gguf::commands][INFO] Required: 4207141728 bytes
...
[2026-04-01][06:19:27][tauri_plugin_llamacpp::commands][INFO] [llamacpp] srv    load_model: loading model '\\?\C:\Users\836D~1\AppData\Roaming\Jan\data\llamacpp\models\JAN-V3~1.5-4\MODEL~1.GGU'
[2026-04-01][06:19:27][tauri_plugin_llamacpp::commands][INFO] [llamacpp] llama_model_load_from_file_impl: using device CUDA0 (NVIDIA GeForce GTX 650) (0000:03:00.0) - 824 MiB free
...
[2026-04-01][06:19:32][tauri_plugin_llamacpp::commands][INFO] [llamacpp] ggml_backend_cuda_buffer_type_alloc_buffer: allocating 2645.91 MiB on device 0: cudaMalloc failed: out of memory
[2026-04-01][06:19:32][tauri_plugin_llamacpp::commands][INFO] [llamacpp] alloc_tensor_range: failed to allocate CUDA0 buffer of size 2774439424
[2026-04-01][06:19:32][tauri_plugin_llamacpp::commands][INFO] [llamacpp] llama_model_load: error loading model: unable to allocate CUDA0 buffer
[2026-04-01][06:19:32][tauri_plugin_llamacpp::commands][INFO] [llamacpp] llama_model_load_from_file_impl: failed to load model
[2026-04-01][06:19:33][tauri_plugin_llamacpp::commands][INFO] [llamacpp] common_init_from_params: failed to load model '\\?\C:\Users\836D~1\AppData\Roaming\Jan\data\llamacpp\models\JAN-V3~1.5-4\MODEL~1.GGU'
[2026-04-01][06:19:33][tauri_plugin_llamacpp::commands][INFO] [llamacpp] srv    load_model: failed to load model, '\\?\C:\Users\836D~1\AppData\Roaming\Jan\data\llamacpp\models\JAN-V3~1.5-4\MODEL~1.GGU'
[2026-04-01][06:19:33][tauri_plugin_llamacpp::commands][INFO] [llamacpp] srv   operator (): operator (): cleaning up before exit...
[2026-04-01][06:19:33][tauri_plugin_llamacpp::commands][INFO] [llamacpp] main: exiting due to model loading error
[2026-04-01][06:19:33][tauri_plugin_llamacpp::commands][ERROR] llama.cpp exited with error code ExitStatus(ExitStatus(1))
```

## Operating System
- [ ] MacOS
- [x] Windows
- [ ] Linux

--- a/src-tauri/plugins/tauri-plugin-llamacpp/src/backend.rs
+++ b/src-tauri/plugins/tauri-plugin-llamacpp/src/backend.rs
@@ -525,9 +525,6 @@ pub async fn prioritize_backends(
         ]
     } else {
         vec![
-            "cuda-cu13.0",
-            "cuda-cu12.0",
-            "cuda-cu11.7",
             "common_cpus",
             "avx512",
             "avx2",

